tembo-stacks/src/apps/pganalyze.yaml

```diff
@@ -0,0 +1,9 @@
+name: !pganalyze
+appServices:
+  - image: quay.io/pganalyze/collector:v0.53.0
+    name: pganalyze-collector
+    env:
+      - name: DB_URL
+        valueFromPlatform: ReadWriteConnection
+      - name: PGA_API_KEY
+        value: ""
```

Comment:
@ChuckHend We're going to be collecting the `PGA_API_KEY` value from users when they activate this app. I'm assuming we'll be able to pass that down through here. Is that true? Is it appropriate to set that value as `""` in this template?

Reply:
Yes the value can be passed down, but it will need to be handled in the control-plane. Nothing in the operator requires the env var to be declared here as empty. So I would say only set this unless the application requires it. 

OSS users will set that directly in the spec. In Tembo cloud thats handled in the control-plane.

Reply:
This one is required by the app, so maybe we leave it here as an indicator / reminder that it needs to be overridden by control plane

Reply:
It might be easier to troubleshoot shoot if this is not provided rather than being assigned an empty string. Just thinking about how those logs will end up looking. There's no way to differentiate user not providing a value vs user providing an empty value, the way this is currently set up.